python2/pow.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python2
+import sha
+import sys
+
+def main():
+	if len(sys.argv) != 4:
+		sys.stderr.write('Usage: %s yourname domain pattern\n' % sys.argv[0])
+		sys.exit(1)
+
+	email = "%s.%s@%s" % (sys.argv[1], "%d", sys.argv[2])
```

Comment:
Hey @jpentland thanks for the pull request. The `.` in `"%s.%s@%s"` will generally make a different email account than the one you are using. Look in to the `+` (plus sign) with email addresses. https://support.google.com/mail/answer/12096?hl=en